cmd/kubeturbo/app/kubeturbo_builder.go

```diff
@@ -1009,3 +1011,47 @@ func reviewSCCAccess(namespace string, kubeClient kubernetes.Interface) bool {
 
 	return true
 }
+
+func WatchConfigMap() {
+	//Check if the /etc/kubeturbo/dynamic-turbo.config exists
+	dynamicConfigfilePath := "/etc/kubeturbo"
+	dynamicConfigfileName := "turbo-dynamic.config"
+
+	viper.AddConfigPath(dynamicConfigfilePath)
+	viper.SetConfigType("json")
+	viper.SetConfigName(dynamicConfigfileName)
+	for {
+		verr := viper.ReadInConfig()
+		if verr == nil {
+			break
+		} else {
+			glog.V(5).Infof("Can't find the dynamic config file %s/%s, will retry in 3 seconds", dynamicConfigfilePath, dynamicConfigfileName)
+			time.Sleep(3 * time.Second)
+		}
```

Comment:
We don't have V(5) logging, the maximum is V(4).

I have a generic question. If the dynamic config does not exist initially when `kubeturbo` starts, but user adds that in the configmap later on, does `kubeturbo` need to restart in order to pick up this file?

Reply:
no need to restart, that's why I use a `for` loop to keep checking if the `autorealod-config` exists or not.  Once it's added, the new logging level will be picked up on the fly.